pkg/models/shortUrl.go

```diff
@@ -0,0 +1,47 @@
+package models
+
+import (
+	"errors"
+)
+
+var (
+	ErrShortURLNotFound = errors.New("Short URL not found")
```

Comment:
[Error strings should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings):
```suggestion
	ErrShortURLNotFound = errors.New("short URL not found")
```